index.mjs

```diff
@@ -179,7 +179,7 @@ const tsConfigurations = [
 
       // typeof any === "evil".
       '@typescript-eslint/no-explicit-any': 'error',
-
+      '@typescript-eslint/prefer-regexp-exec': 'off', // RegExp.exec() is slightly faster than String.match() so will leave this off and not worth updating our code
```

Comment:
update comment to `is *only* slightly faster` to make it clear why we're disabling